dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/utils/OpenAPITestUtil.java

```diff
@@ -277,6 +279,19 @@ public static void checkPaths(JsonNode root,
         assertThat("Path names", pathNames, hasSize(expectedCount));
     }
 
+    public static void checkServerContextRoots(JsonNode root, String expectedContextRoot) {
```

Comment:
This is called contextRoots but only takes a singular root. Could be fixed in either direction.

Reply:
It checks every server's path has the expected context root, so multiple servers are checked for the same context root.

I guess `checkServersForContextRoot` would be clearer.

Reply:
Changed the name and added some doc.